migrate.py

```diff
@@ -98,6 +100,8 @@ def make_blockquote(text):
 class Migrator():
     def __init__(self, trac_url, github_username=None, github_password=None, github_project=None,
                  github_api_url=None, username_map=None, config=None):
+        if trac_url[-1]!='/':
+            trac_url=trac_url+'/'
```

Comment:
handle trac projects that are not at the root of the host, e.g. http:/my.trac.host.com/myTracProject